src/components/Form/Telephone/Telephone.jsx

```diff
@@ -122,131 +76,118 @@ export default class Telephone extends ValidationElement {
       number: this.getFormattedNumber(),
       extension: this.props.extension,
       noNumber: this.props.noNumber || false,
-      ...queue
+      ...queue,
     })
   }
 
-  updateNumber(type, values) {
-    this.update({ type, noNumber: false })
-  }
-
-  updateToggleDomestic() {
-    this.setState({ ...defaultNumbers }, () => {
-      this.updateNumber('Domestic')
+  updateNumber = (type) => {
+    this.update({
+      type,
+      noNumber: false,
     })
   }
 
-  updateToggleDsn() {
-    this.setState({ ...defaultNumbers }, () => {
-      this.updateNumber('DSN')
+  updateToggleDomestic = () => {
+    this.setState({
+      ...defaultNumbers,
+    }, () => {
+      this.updateNumber('Domestic')
     })
   }
 
-  updateToggleInternational() {
-    this.setState({ ...defaultNumbers }, () => {
+  updateToggleInternational = () => {
+    this.setState({
+      ...defaultNumbers,
+    }, () => {
       this.updateNumber('International')
     })
   }
 
-  updateDsnFirst(values) {
-    const dsn = { ...this.state.dsn, first: values.value }
-    this.setState({ dsn }, () => {
-      this.updateNumber('DSN')
-    })
-  }
-
-  updateDsnSecond(values) {
-    const dsn = { ...this.state.dsn, second: values.value }
-    this.setState({ dsn }, () => {
-      this.updateNumber('DSN')
-    })
-  }
-
-  updateDomesticFirst(values) {
-    const domestic = { ...this.state.domestic, first: values.value }
-    this.setState({ domestic }, () => {
+  updateDomesticFirst = (values) => {
+    this.setState({
+      first: values.value,
```

Comment:
@christopherhuii I think these update functions still need to set `...this.state.domestic` etc because when I tried testing this and entering a value, nothing actually appears in the input.